memgpt/metadata.py

```diff
@@ -640,6 +640,7 @@ def add_persona(self, persona: PersonaModel):
         with self.session_maker() as session:
             session.add(persona)
             session.commit()
+            return persona
```

Comment:
should add type annotation:

```python
def add_persona(self, persona: PersonaModel) -> PersonaModel:
```

Also do the same (add return + return type) for `add_human`?